pages/common/texdoc.md

```diff
@@ -0,0 +1,16 @@
+# texdoc
+
+> Search for appropriate documentation for (La)TeX commands or packages.
+> More information: <https://texdoc.org/index.html>.
+
+- Open first search result in the default PDF viewer:
+
+`texdoc {{search}}`
+
+- List best search result:
+
+`texdoc --list {{search}}`
+
+- Open full documentation of texdoc:
```

Comment:
Perhaps this will read better?
```suggestion
- Open the full documentation for a command or package:
```

Does this change the meaning at all here?

Reply:
I already showed how to open any documentation in example 1. I thought I should add the command to open the documentation of texdoc with texdoc for more information.

Reply:
Then I think it's okay to keep it here. 👍🏻